lib/modules/manager/maven-wrapper/artifacts.ts

```diff
@@ -150,13 +154,27 @@ async function executeWrapperCommand(
   };
 
   try {
-    await exec(cmd, execOptions);
+    let cmdToExecute = cmd;
+    const customArtifactoryUrl = getCustomMavenWrapperUrl(deps);
+    if (customArtifactoryUrl) {
+      cmdToExecute = `export MVNW_REPOURL=${customArtifactoryUrl} && ${cmd}`;
```

Comment:
no, add to extra ENV of exec options instea

Reply:
thanks, 
implemented, do you have any tips how to unit test that scenario ?